packages/node/src/lib/ReadMe.ts

```diff
@@ -197,8 +201,9 @@ export default class ReadMe {
         try {
           const webhookData = await testVerifyWebhook(baseUrl, email, this.readmeProjectData.jwtSecret as string);
           return res.json({ ...webhookData });
-        } catch (e) {
-          return res.status(400).json({ error: (e as Error).message });
+        } catch (err) {
+          logger.error({ err, message: 'Test verify webhook error' });
```

Comment:
```suggestion
          logger.error({ err, message: 'Webhook verification failed.' });
```

@erunion thoughts on including documentation URLs with our logger class? I think breaking out doc links into a separate key may be more consumable.

Reply:
agreed that we should give the user as much information as we can to help them resolve their issues.